### DISCUSSION TITLE
Disable a new collections

### DISCUSSION THREAD

**Discussion by urmauur** [tinacms/tinacms#2805] [Help]
![Screen Shot 2022-04-13 at 23 45 12](https://user-images.githubusercontent.com/10354610/163229909-dcbaf470-e04c-4ba5-bb41-4853f2768557.png)

Hi guys, is possible to disable create new collections from admin?

**Comment by jeffsee55** [comment]
@urmauur it's not currently possible, but that sounds like something we can make configurable!

**Reply by urmauur** [reply]
> 

can you send me an example how? or just waiting the next release? yeah sometime we just need a data without `create new`

**Reply by jeffsee55** [reply]
It wouldn't be possible yet, we need to add this ability through a configuration that we expose. We'll discuss this as a feature and update this discussion if/when we plan to move forward with it.

**Reply by urmauur** [reply]
> It wouldn't be possible yet, we need to add this ability through a configuration that we expose. We'll discuss this as a feature and update this discussion if/when we plan to move forward with it.

Thank you very much for fast response :)

**Reply by urmauur** [reply]
@jeffsee55 some suggestion maybe we can add props on the future eg `create: true | false`

**Comment by Perni1984** [comment]
we would also have a need for this feature where we only have one document inside the collection (as the structure differs a lot between content types).

Moreover it would be really helpful to be able to configure the admin to directly open contextual editing, if the collection is configured to contain only one document.

**Reply by scottgallant** [reply]
> Moreover it would be really helpful to be able to configure the admin to directly open contextual editing, if the collection is configured to contain only one document.

Would you want to configure which route gets viewed?

**Reply by Perni1984** [reply]
yes, but we can control that via the RouteMappingPlugin I guess.

**Comment by scottgallant** [comment]
Maybe what we really need is a Single Document Collection, as described here:  [](https://github.com/tinacms/tinacms/discussions/2948), assuming it would not allow you to create more entries of that type. 

Tina isn't really set up to deal with content that is not intended to have multiple entries (like a site header or site footer).

**Comment by logan-anderson** [comment]
The ability to hide the delete and create buttons is implemented in this PR: https://github.com/tinacms/tinacms/pull/3349
